src/SDKs/ManagedServiceIdentity/Management.ManagedServiceIdentity/Generated/ManagedServiceIdentityClient.cs

```diff
@@ -287,7 +328,7 @@ private void Initialize()
             Operations = new Operations(this);
             UserAssignedIdentities = new UserAssignedIdentitiesOperations(this);
             BaseUri = new System.Uri("https://management.azure.com");
-            ApiVersion = "2015-08-31-preview";
```

Comment:
@wendyNEU are you planning to release a stable version for your api version `2015-08-31-preview` ?

Reply:
I plan to release a stable version for api version 2018-11-30